content/docs/testing-farm.md

```diff
@@ -60,6 +60,16 @@ jobs:
   - fedora-latest-stable
 ```
 
+If you want to trigger tests via pull request comment and not by every new commit into the pull request, the test job needs to include `manual_trigger` (described below) option in the job configuration:
+```yaml
+  jobs:
+  - job: tests
+    trigger: pull_request
+    targets:
+    - fedora-all
+    manual_trigger: true
```

Comment:
```suggestion
    skip_build: true
    manual_trigger: true
```

just for clarity, since without `skip_build` enabled, the build job needs to be configured as well